docs/configuration-reference/components/velero.md

```diff
@@ -37,6 +37,7 @@ Velero component configuration example:
 # velero.lokocfg
 component "velero" {
 
+  #provider = "azure/openebs/restic"
```

Comment:
I don't know but no space here seems a little bit weird.

Reply:
What space you mean? Blank line separating parameter from block below?

Reply:
Yes, I meant `# provider` as we are using it in line with provider name in test file.